### PR TITLE
Slugifies People

### PR DESCRIPTION
**Pull request by BriocheBerlin**

### CODE REVIEW

File: app/models/person.rb [hunk 1]
```diff
@@ -42,6 +42,22 @@ class Person < ActiveRecord::Base
...
+    [
+      :full_name,
+      [:full_name, rand(1000)]
+    ]
+  end
+
+
+  def full_name
+    "#{first_name} #{last_name}"
+  end
```
Comment (til) [inline app/models/person.rb]: It seems `full_name` now exists twice in this file.

File: db/migrate/20150929181908_add_slug_to_people.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class AddSlugToPeople < ActiveRecord::Migration
+  def change
+		add_column 	:people, :slug, :string
+	  add_index 	:people, :slug, unique: true
+  end
+end
```
Comment (til) [inline db/migrate/20150929181908_add_slug_to_people.rb]: Indentation.

Comment (til) [inline db/migrate/20150929181908_add_slug_to_people.rb]: ... is a bit wonky.

File: app/models/person.rb (hunk 1 above)
Comment (til) [inline app/models/person.rb]: Ha, and a spec caught it

**Comment by til** [comment]
I don't know how much work it would be to get this working with the current version of master - @BriocheBerlin would you be fine with us closing it, or do you want to give it a look? I know you have time, because you are in holidays now ... scnr 😈.

**Comment by sareg0** [comment]
@BriocheBerlin I'm just gonna close this PR up, for now. But maybe one day, we will indeed slugify all the people :) Thank you for you work here!

**Comment by til** [comment]
slug

**Pull request closed without merging**